.github/workflows/translation_check.py

```diff
@@ -0,0 +1,177 @@
+"""
+This script is triggered after a new PR is created on OpenRCT2 Localisation repository.
+It checks the difference in translation count between master and PR branch.
+The results is put to result.md to be later published as a comment.
+"""
+
+import filecmp
+import os
+import re
+
+languages = []
+
```

Comment:
```suggestion
MASTER_LANG_DIR = "master/data/language"
PR_LANG_DIR = "master/data/language"

```
Define these as global constants and reuse elsewhere in the code, so it becomes easier to change later if needed.